src/EventListener/InitializeSystemListener.php

```diff
@@ -151,6 +167,9 @@ private function boot($routeName, $basePath)
         // Log PHP errors
         $this->iniSet('error_log', $this->rootDir . '/system/logs/error.log');
 
+        // Support legacy Session access
+        $this->initializeLegacySessionAccess();
```

Comment:
Then may I ask you to move it down to where it used to be? We should not mess with the initialization order too much, because we cannot foresee the effects.